File: source/kCura.Relativity.Export.ConvertedToC#/Exporters/LoadFileCellFormatters/ILoadFileCellFormatter.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
using kCura.Relativity.Export.Exports;

namespace kCura.Relativity.Export.Exports
{
	public interface ILoadFileCellFormatter
	{
		string TransformToCell(string contents);
		string CreateNativeCell(string location, ObjectExportInfo artifact);
		string CreateImageCell(ObjectExportInfo artifact);

		string RowPrefix { get; }
		string RowSuffix { get; }
	}






}

File: source/kCura.Relativity.Export/Exporters/LoadFileCellFormatters/ILoadFileCellFormatter.vb
Imports kCura.Relativity.Export.Exports

Namespace kCura.Relativity.Export.Exports
	Public Interface ILoadFileCellFormatter
		Function TransformToCell(ByVal contents As String) As String
		Function CreateNativeCell(ByVal location As String, ByVal artifact As ObjectExportInfo) As String
		Function CreateImageCell(ByVal artifact As ObjectExportInfo) As String

		ReadOnly Property RowPrefix() As String
		ReadOnly Property RowSuffix() As String
	End Interface





	
End Namespace

